spec/index.md

```diff
@@ -697,6 +705,7 @@ canister developers that aim at keeping data confidential are advised to add a s
 All requested paths must have the following form:
 
 -   `/time`. Can always be requested.
+- `/api_boundary_nodes`, `/api_boundary_nodes/<node_id>`, `/api_boundary_nodes/<node_id>/domain`, `/api_boundary_nodes/<node_id>/pub_key`, `/api_boundary_nodes/<node_id>/ip4_address`, `/api_boundary_nodes/<node_id>/ipv6_address`. Can always be requested.
```

Comment:
Two questions:
1. Is the api_boundary_nodes information replicated in the state tree of all subnets or just in root subnet?
2. Is the request sent to /api/v2/canister/<effective_canister_id>/read_state`  or `/api/v2/subnet/<subnet_id>/read_state`? And which effective_canister_id/subnet_id? This will decide which subnet handles the request.

Reply:
I believe the information should be replicated across subnets for the sake of load balancing.

Reply:
Please see the [slides](https://docs.google.com/presentation/d/1TyHR0uRiHZsXSTWsuMjnlWJ5DP0hu8C6afaGXGDKMjE/edit#slide=id.g29b6a5284b2_0_523) for the visualization of depth calculation for `/api_boundary_nodes`
Here are the results:

- `/api_boundary_nodes` make `/subnet` one more layer deep

- In mainnet topology,  the depth of `/subnet` increases from 22 to 23

- `/api_boundary_nodes` has the max depth of 15 if there are fewer than 64 api boundary nodes